mtp_api/apps/mtp_auth/tests/test_views.py

```diff
@@ -52,6 +53,61 @@ def prison_set(user):
     )
 
 
+class JobInformationTestCase(APITestCase, AuthTestCaseMixin):
+    fixtures = [
+        'initial_groups.json',
+        'initial_types.json',
+        'test_prisons.json',
+    ]
+
+    url = reverse('job-information-list')
```

Comment:
if you're going to have this at class-level, you ought to use the `reverse_lazy` form, because urls can be dynamic. it doesn't make any difference in this case, but would be good practice

Reply:
`reverse_lazy` only looks up the url when the value is _accessed_ not at the moment it's called